https://issues.redhat.com/browse/ACM-13133 (#6890)
open-cluster-management/rhacm-docs: governance/create_config_pol.adoc

*Required access*: Administrator or cluster administrator

* <<creating-a-configuration-policy,Creating a configuration policy>>

> * <<disabling-configuration-policies,Disabling configuration policies>>

Author: ```suggestion
* <<viewing-your-configuration-policy-from-the-console,Viewing your configuration policy from the console>>
* <<disabling-configuration-policies,Disabling configuration policies>>
```

* <<deleting-a-configuration-policy,Deleting a configuration policy>>

[#creating-a-configuration-policy]
== Creating a configuration policy

> You can create a YAML file for your configuration policy from the command line interface (CLI) or from the console. As you create a configuration policy from the console, a YAML file is also created in the YAML editor.

Author: ```suggestion
You can create a YAML file for your configuration policy from the command line interface (CLI) or from the console. As you create a configuration policy from the console, a YAML file is displayed in the YAML editor.
```

*Required access*: Administrator or cluster administrator

* <<creating-a-configuration-policy,Creating a configuration policy>>

> * <<viewing-your-configuration-policy-from-the-console,Viewing your configuration policy from the console>>

Author: ```suggestion
```

Complete the following steps to view your configuration policy from the CLI:

. View details for a specific configuration policy by running the following command:

+
[source,bash]
----
oc get policies.policy.open-cluster-management.io <policy-name> -n <namespace> -o yaml
----

. View a description of your configuration policy by running the following command:

+
[source,bash]
----
oc describe policies.policy.open-cluster-management.io <name> -n <namespace>
----

[#creating-a-configuration-policy-from-the-console]
=== Creating a configuration policy from the console

As you create a configuration policy from the console, a YAML file is also created in the YAML editor.

. Log in to your cluster from the console, and select *Governance* from the navigation menu.

. Click *Create policy*. Specify the policy you want to create by selecting one of the configuration policies for the specification parameter. 

. Continue with configuration policy creation by completing the policy form. Enter or select the appropriate values for the following fields:
  - Name
  - Specifications
  - Cluster selector
  - Remediation action
  - Standards
  - Categories
  - Controls

> . Click *Create*. Your configuration policy is created.

Author: Part of the doc effort is to remove console steps.

Select one of your policies to view more details. The _Details_, _Clusters_, and _Templates_ tabs are displayed.

[#disabling-configuration-policies]
== Disabling configuration policies

To disable your configuration policy, select *Disable policy* from the _Actions_ menu of the policy. The policy is disabled, but not deleted.

[#deleting-a-configuration-policy]
== Deleting a configuration policy

Delete a configuration policy from the CLI or the console. Complete the following steps:

. To delete the policy from your target cluster or clusters, run the following command:

+
[source,bash]
----
oc delete policies.policy.open-cluster-management.io <policy-name> -n <namespace>
----

. Verify that your policy is removed by running the following command:

+
[source,bash]
----
oc get policies.policy.open-cluster-management.io <policy-name> -n <namespace>
----

> . To delete a configuration policy from the console, click the *Actions* icon for the policy you want to delete in the policy violation table and then click *Remove*. 

Reviewer: Should be "Delete" 
![Screenshot 2024-09-09 at 5 57 38 PM](https://github.com/user-attachments/assets/0b7a2f9b-a037-40c1-b272-b2d8ecd507b8)

Reviewer: ![Screenshot 2024-09-09 at 6 02 00 PM](https://github.com/user-attachments/assets/7c1e66f9-eb9e-471b-a989-8ddd4c283fe0)

Reviewer: LGTM

> . To delete a configuration policy from the console, click the *Actions* icon for the policy you want to delete in the policy violation table and then click *Remove*. 

Author: ```suggestion
. To delete a configuration policy from the console, click the *Actions* icon for the policy you want to delete in the policy violation table and then click *Delete*. 
```
